conda/meta.yaml

```diff
@@ -5,24 +5,20 @@ package:
   version: {{ version }}
 
 source:
-  git_url: https://github.com/Theano/Theano.git
-  git_tag: rel-{{ version }}
+  path: ../
 
 build:
   noarch: python
   script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
   build:
-    - m2-filesystem    [win]
-    - m2-git           [win]
-    - git              [not win]
     - python
     - setuptools
     - six >=1.9.0
     - numpy >=1.9.1
     - scipy >=0.14.0
-    - pygpu >=0.6,<0.7
+    - pygpu >=0.7
```

Comment:
should we put <0.8?

Reply:
Actually we could remove this since we don't need anything for the build (except setuptools and python).